app/javascript/menu/main-menu.jsx

```diff
@@ -53,6 +54,7 @@ export const MainMenu = ({
   useEffect(() => {
     // cypress, debugging
     window.ManageIQ.menu = menu;
+    console.log("event.target-----------",document.activeElement);
```

Comment:
You probably don't want this in your final code...

Reply:
yes @skateman i added a line in description that there is a bug i will work on it and will remove console logs then.

Reply:
Removed the console log